Package.swift

```diff
@@ -3,7 +3,7 @@ import PackageDescription
 
 let package = Package(
     name: "Orbit",
-    platforms: [.iOS(.v13)],
+    platforms: [.iOS(.v14)],
```

Comment:
We don't want to raise the iOS minimal version for all clients just because of the secure version of this component. We will probably raise it when absolutely necessary in future.

Let's only add an `if available` branch and add a new additional init so that the init for `isSecure` is available only for >=14.